client/src/components/DataFiles/DataFilesSystemSelector/DataFilesSystemSelector.jsx

```diff
@@ -17,7 +17,10 @@ const DataFilesSystemSelector = ({
     (state) => state.systems.storage.configuration
   );
   const modalProps = useSelector((state) => state.files.modalProps[operation]);
-  const findSystem = (id) => systemList.find((system) => system.system === id);
+  const findSystem = (id) =>
```

Comment:
> const findSystem = (id) =>

Should we rename the `id` param here to be more representative of the new string sent by the dropdown?

Reply:
Yeah I think that would help with some clarity. Similarly, I think it would also be useful to change the `systemId` prop in this [component](https://github.com/TACC/Core-Portal/blob/2091094150bbd1f7a4bdd29532af7b4b72162ca1/client/src/components/DataFiles/DataFilesSystemSelector/DataFilesSystemSelector.jsx#L8) be more descriptive as well since it also contains the `system + homeDir` value. Though this would be a bigger change since this component is used elsewhere as well

Reply:
Can you add a todo for that under the files epic? We'll move that to general portal work later

Reply:
Created task [TV3-123](https://jira.tacc.utexas.edu/browse/TV3-123)